Create Hall_of_jail.md

FLYEMOJ1 opened the pull request:
添加Hall of Jail.
用于记录违反TOS并被拒绝提供服务的用户。

FLYEMOJ1 committed:
Create Hall_of_jail.md

添加Hall of Jail.
用于记录违反TOS并被拒绝提供服务的用户。

FLYEMOJ1 commented:
尚未完成。
需讨论图片保存位置。

SerinaNya reviewed (changes requested):
请参照中文排版指北修改此文档

FLYEMOJ1 commented:
> 请参照中文排版指北修改此文档

望能修改部分内容让我能理解去修改哪些部分。

SerinaNya commented:
@FLYEMOJ1 英文左右加空格

FLYEMOJ1 commented:
@jinzhijie 左右加空格不太美观

FLYEMOJ1 committed:
Update Hall_of_jail.md

参照中文排版指北修改部分内容。
感谢 @jinzhijie @tnqzh123 进行的指正。

FLYEMOJ1 commented:
是我本人理解错误。
已做修改。

FLYEMOJ1 committed:
Add Hall_of_jail_image

添加 Hall of jail的图片信息。
以 U@[plantform] 的方式继续添加
并添加内容至 .gitignore + .DS_Store

FLYEMOJ1 commented:
以LSCommspt-bot的image储存方式保存。
请注意命名。

FLYEMOJ1 commented:
@jinzhijie 检查提交信息，请（半恼

Qubik65536 reviewed (changes requested):
根据标准命名文件并将图片文件放入 `assets` 或 `assets/images` 文件夹，向文件内加入图片。

------

其他看起来都没问题

FLYEMOJ1 committed:
Rename Hall_of_jail_image to Assets

根据LSCommspt-bot的image存储标准进行保存。
https://github.com/LittleSkinCommspt/commspt-bot-manual/pull/16#pullrequestreview-926957765

Qubik65536 reviewed (changes requested):
然后向文章内加入图片/图片链接？我觉得没别的问题了

FLYEMOJ1 commented:
已更新。
最近在忙别的计划，望诸位有时间可以帮我继续完善该文档。

Qubik65536 commented:
@Restent @jinzhijie

SerinaNya reviewed (changes requested):
图片确实是上传了，但是文档里没写呀

Cubik65536 commented:
我会在 UTC 9:30 PM 左右提交新的 pull request

Qubik65536 commented:
#17 完成了剩下的工作
